source/solvers/gls_navier_stokes.cc

```diff
@@ -89,12 +86,12 @@ GLSNavierStokesSolver<dim>::setup_dofs_fd()
             BoundaryConditions::BoundaryType::noslip)
           {
             VectorTools::interpolate_boundary_values(
-              mapping,
+              *this->velocity_mapping,
```

Comment:
You should keep only a single mapping for both variables. Be careful to use it everywhere.

Reply:
The pressure mapping was removed and velocity mapping renamed to "mapping"